### ISSUE TITLE
About focal loss

### ISSUE THREAD

**Issue by filaPro** [strongwolf/DW#10]
Hi,

Thanks for your interesting research. I have a question about focal loss in formula (12) of the paper. Looks like in the [code](https://github.com/strongwolf/DW/blob/main/dw_head.py#L175) you simply use `F.binary_cross_entropy` instead of focal loss. Am I right? And do you have any specific reasons for it?

**Comment by strongwolf** [comment]
Hi, The term p_neg_weight already consists of the scaling factor joint_conf**2, which is the same as in the focal loss, so there is no difference.
